packages/odata2ts/test/generator/v2/EntityBasedGenerationTests.ts

```diff
@@ -164,6 +164,24 @@ export function createEntityBasedGenerationTests(
     });
   });
 
+  test(`${testSuiteName}: base class order`, async () => {
```

Comment:
This test is not necessary. Inheritance works the same in V2 and V4. So I would suggest to remove this test. Or did I miss anything?